src/main.ts

```diff
@@ -5,6 +5,11 @@ import { IConfigurationService } from '@/config/configuration.service.interface'
 async function bootstrap(): Promise<void> {
   const app = await new DefaultAppProvider().provide(AppModule.register());
 
+  app.enableCors({
+    origin: 'http://localhost:3000', // your frontend origin
+    credentials: true, // allows cookies or other credentials
+  });
+
```

Comment:
Need to remove this before merging. Maybe we can find a way to have this enabled on local dev